code/modules/client/preferences_savefile.dm

```diff
@@ -170,7 +170,10 @@
 	S["gen_record"]			>> gen_record
 	S["be_special"]			>> be_special
 	S["disabilities"]		>> disabilities
-	S["player_alt_titles"]		>> player_alt_titles
+	S["player_alt_titles"]	>> player_alt_titles
+	S["used_skillpoints"]	>> used_skillpoints
+	S["skills"]				>> skills

```

Comment:
This is pointless. We don't use the savefile at all, everything is handled via the SQL database, which would require a schema update.